src/Util/Emailer.php

```diff
@@ -72,20 +72,28 @@ public static function send($params)
 		$multipartMessageBody .=
 			"--" . $mimeBoundary . "--\n";					// message ending
 
+		if (Config::get("system", "sendmail_params", true)) {
```

Comment:
Can you please add this new htconfig setting to `doc/htconfig.md` ?

Reply:
I would have preferred it to be tied to another settings where you can select your email provider but we don't have that at the moment, so this is low impact and fine as Sendmail is the default on many system, I believe.